### ISSUE TITLE
Rounding flags

### ISSUE THREAD

**Issue by c3d** [c3d/DB48X-on-DM42#1131]
Add rounding flags for decimal computations, to be able to round up, down or towards/away from zero.

**Comment by Wiljea** [comment]
Are these rounding flags what would allow to implement the rounding operations +inf +sup *inf *sup which are suggested in the French article on interval arithmetic: https://fr.wikipedia.org/wiki/Arithm%C3%A9tique_d%27étapes? Furthermore, are these functions currently defined? And, if so, how are they defined or used ?

**Comment by c3d** [comment]
No, the function are not implemented yet, as indicated by the fact that the issue is still open. Furthermore, a quick study of what it would take to implement them highlighted that it was not that simple.
